front/poke/temporal/activities.ts

```diff
@@ -394,9 +393,11 @@ export async function deleteAppsActivity({
       });
       logger.info(`[Workspace delete] Deleting app ${app.sId}`);
       await app.destroy({ transaction: t });
-    }
+    });
+  }
 
-    await KeyResource.deleteAllForWorkspace(workspace);
+  await frontSequelize.transaction(async (t) => {
```

Comment:
Why not keeping the same transaction as above?

Reply:
The transaction above deletes one app and its runs all together, in a loop, which at each iteration calls `coreAPI.deleteProject()` (outside of the transaction now).